src/Api/Controllers/GroupsController.cs

```diff
@@ -38,13 +41,20 @@ public class GroupsController : Controller
         _createGroupCommand = createGroupCommand;
         _updateGroupCommand = updateGroupCommand;
         _deleteGroupCommand = deleteGroupCommand;
+        _authorizationService = authorizationService;
     }
 
     [HttpGet("{id}")]
     public async Task<GroupResponseModel> Get(string orgId, string id)
```

Comment:
nit: Switch both of these to being `Guid` then you no longer need to create the guid yourself.